pkg/cmd/roachtest/liquibase.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package main
+
+import "context"
+
+// This test runs the Liquibase test harness against a single cockroach node.
+
+func registerLiquibase(r *testRegistry) {
+	runLiquibase := func(
+		ctx context.Context,
+		t *test,
+		c *cluster,
+	) {
+		if c.isLocal() {
+			t.Fatal("cannot be run in local mode")
+		}
+		node := c.Node(1)
+		t.Status("setting up cockroach")
+		c.Put(ctx, cockroach, "./cockroach", c.All())
+		c.Start(ctx, t, c.All())
+
+		version, err := fetchCockroachVersion(ctx, c, node[0], nil)
+		if err != nil {
```

Comment:
nit: I've been using `require.NoError(t, err)` for these, do you think it's better to use these in the future?